applications/NXfluo.nxdl.xml

```diff
@@ -27,6 +27,15 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd "
     >
+    <symbols>
+      <doc>
+        The symbol(s) listed here will be used below to coordinate datasets
+        with the same shape.  
+      </doc>
+      <symbol name="nE">
+        <doc>number of energies</doc>
```

Comment:
Capitalize here and below

Reply:
Sorry Peter I am not understanding you, do you mean that "number of energies' should be "Number of Energies"?